wallet/i18n/de.json

```diff
@@ -1,6 +1,6 @@
 {
   "app_title": "Pylonen-Geldbörse",
-  "something_wrong": "Etwas ist schief gelaufen",
+  "something_wrong": "Etwas ist schief gelaufen, Probeer de app opnieuw te starten.",
```

Comment:
This translation has to be fixed, please use German, this is Dutch. i.e. Probieren is German, Probeer is Dutch.